### PR TITLE
Document how to disable some labextensions with config

### PR DESCRIPTION
**Pull request by yuvipanda**
Porting this blog post into the docs:
https://words.yuvi.in/post/jupyterlab-settings-z2jh/

### CODE REVIEW

**Comment by yuvipanda** [comment]
The failing linkcheck is unrelated?

image

File: doc/source/jupyterhub/customizing/user-environment.md [hunk 1]
```diff
@@ -541,3 +541,43 @@ Prior to 2.0, the default behavior of zero-to-jupyterhub was to launch `jupyterh
...
+and JupyterLab's [page_config.json](https://jupyterlab.readthedocs.io/en/stable/user/directories.html#labconfig-directories)
+
+JupyterLab's `page_config.json` lets you set page configuration by dropping JSON files
+under a `labconfig` directory inside any of the directories listed when you run `jupyter --paths`.
+We just use `singleuser.extraFiles` to provide this file!
+
+```yaml
+singleuser:
+  extraFiles:
+  lab-config:
```
Comment (consideRatio) [inline doc/source/jupyterhub/customizing/user-environment.md:559]: From mobile, but looks like indentation bug here

Comment (yuvipanda) [reply doc/source/jupyterhub/customizing/user-environment.md]: Ah, fixed @consideRatio

**Review by consideRatio — APPROVED** [review]

**Comment by yuvipanda** [comment]
Thanks a lot, @consideRatio!

**Pull request merged**